roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/BrowserHelper.java

```diff
@@ -461,6 +474,17 @@ public static BrowseFileBundle retrieveBrowseFileBundle(IndexedAIP aip, IndexedR
       bundle.setRepresentationInformationFields(Collections.emptyList());
     }
 
+    ModelService model = RodaCoreFactory.getModelService();
+    DistributedInstance distributedInstance = null;
+    try {
+      distributedInstance = model.retrieveDistributedInstance(aip.getInstanceId());
+    } catch (NotFoundException e) {
+      LOGGER.warn("Could not retrieve the distributed instance", e);
+    } catch (AuthorizationDeniedException e) {
+      LOGGER.warn("Do not have Authorization", e);
+    }
```

Comment:
You need to check the instance type. The local instance does not have this configuration.

Reply:
Done